packages/circuit-ui/components/Body/Body.docs.mdx

```diff
@@ -28,6 +23,104 @@ The Body component comes in two sizes. Use the default `one` size in most cases.
+- Allow users to access the expanded form of abbreviations ([3.1.4: Abbreviations](https://www.w3.org/WAI/WCAG21/Understanding/abbreviations))
+- Write content as clearly and simply as possible ([3.1.5: Reading Level](https://www.w3.org/WAI/WCAG21/Understanding/reading-level))
+
+_Note: the success criteria referenced above are intended for AAA conformance. Though they are not requirements, they are a best practice and following them will improve the accessibility of our content._
+
+#### Complement text with images
+
+Images, graphs, or other illustrations can help users contextualize and/or understand a piece of content.
+
+Visuals should usually complement text—and not replace it—unless an excellent description is provided as alternative text.
+
+#### Translate content
+
+Translate content intended for a linguistically diverse audience, and make it easy for users to change the page's language.
+
+Bear in mind that machine translation is often inaccurate: prefer professional translation in order to give the best possible experience to users from diverse linguistic backgrounds.
+
+#### Do not rely on color alone
+
+Color alone is not sufficient to differentiate text from the surrounding content ([1.4.1 Use of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html)).
+
+This is especially relevant when using the `success` and `error` variants.
+
+For example, imagine a list of good and bad accessibility practices. It is not enough to use the `success` (green) variant for good practices and the `error` (red) variant for bad practices. Instead, use additional visual cues such as icons (with an accessible label), or break up the list into two separate ones with explicit labeling.
+
+#### Make status messages accessible to screen readers using live regions
+
+When a change in content is not given focus (typically a message being added to the DOM to reflect a status update), the change needs to be announced to screen readers using a [live region](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions) ([4.1.3 Status Messages](https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html), [3.3.1 Error Identification](https://www.w3.org/WAI/WCAG21/Understanding/error-identification)).
+
+For example, a form submission fails and a message saying "This username is already taken." using `<Body variant="error" />` appears. Without a live region, screen reader users would have no way of knowing that something happened and that they need to make changes to the form before submitting again. They will either have to step through the DOM to try and find the relevant error message, or they will give up and close the page.
+
+Live regions can be tricky to work with, therefore we recommend using existing components such as the [NotificationInline](Notifications/NotificationInline) or the [NotifiationToast](Notifications/NotificationToast) instead of implementing them from scratch.
```

Comment:
Note: this assumes that these components are already released. I'd leave it here since it will be the case soon and I fear we'll forget to add this later, but happy to take it out if you prefer